packages/formvuelate/src/SchemaFormFactory.js

```diff
@@ -22,6 +22,18 @@ export default function SchemaFormFactory (plugins = [], components = null) {
     }
   }
 
+  const formProps = { ...SchemaForm.props }
+
+  function extendProps (newProps = {}) {
+    Object.assign(formProps, newProps || {})
+  }
```

Comment:
suggestion: Youre checking `|| {}` for a falsey value, but already setting a default of `{}` on the param. If we want to be thorough lets instead check that `newProps` is an object before the `assignment`, maybe throw a console.warn or something, because if someone passes an empty array for example an array `[1,2,3]` it would still work, and would assign the indexes as properties and the array values as values.

Or did you intent for this to be use at your discretion? :D

Reply:
Yea agreed, this was a quick thing to get this working but I could've easily forgotten to fix it. I will do a proper type check before the assignment.